src/config.ts

```diff
@@ -150,6 +152,7 @@ export default {
   PORT: Number(PORT) || 3000,
   POSITRON_API_BASE,
   PREDICTION_ENDPOINT,
+  PRODUCTION_ENV: NODE_ENV === "production",
```

Comment:
We were doing this in enough files that I figured it could go here

Reply:
👍